flake8diff/main.py

```diff
@@ -114,10 +113,33 @@
          'Can be any of "{0}"'
          ''.format(', '.join(STRICT_MODES.keys())),
 )
+parser.add_argument(
+    '--version',
+    action='store_true',
+    default=False,
+    help='Show the version number of flake8-diff'
+)
+
+
+def version():
+    msg = (
+        'flake8-diff\n'
+        '===========\n'
+        '{}\n\n'
```

Comment:
@gregarmer let me know if you think its appropriate to print description here as well. I think it doesnt hurt but not sure if there are any unix standards for these things. current output looks like:

```bash
❯❯❯ flake8-diff --version
flake8-diff
===========
Run flake8 across a set of changed files and filter out violations occurring only on the lines that were changed.

version: 0.2.0
python: /Users/miki725/.virtualenvs/flake8/bin/python3.4
source: https://github.com/dealertrack/flake8-diff
```